### DISCUSSION TITLE
Mongodb aggregate

### DISCUSSION THREAD

**Discussion by Valdri** [Budibase/budibase#6395] [Help] [answered]
Hi,
I cannot find aggregate in the action types.
Does budibase support mongodb aggregate?

**Comment by melohagan** [comment, marked as answer]
Hi @Valdri 

Not at the moment, no. There's quite a number of collection methods that's aren't yet supported:
https://www.mongodb.com/docs/v4.4/reference/method/js-collection/

What you see is what you get for now!

There's an open issue (https://github.com/Budibase/budibase/issues/5387) to expand and improve upon the current MongoDB connector.

**Comment by flpms** [comment]
So sad to see this answer, so it's look like retool sees a better option for MongoDB users.

**Reply by melohagan** [reply]
I have been working on this: https://github.com/Budibase/budibase/pull/8117

Having looked at Retools MongoDB integration, it does look to be a good closed-source option.
